disposable_email_blacklist.conf

```diff
@@ -1544,6 +1560,7 @@ secretemail.de
 secure-mail.biz
 secure-mail.cc
 secured-link.net
+securehost.com.es
```

Comment:
This one does not seem as a disposable address host (http://securehost.com.es/)
Where are you getting these?